app/src/main/java/com/go/playlistmaker/App.kt

```diff
@@ -0,0 +1,37 @@
+package com.go.playlistmaker
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
+
+const val PLAYLIST_MAKER_PREFERENCES = "playlist_maker_preferences"
+const val LIGHT_THEME_KEY = "light"
+
+class App : Application() {
+
+    var darkTheme = false
```

Comment:
Данную переменную лучше сделать приватной, либо хотя бы сделать ей приватный сеттер:
```suggestion
    var darkTheme = false
        private set
```
Таким образом, любые другие классы снаружи не смогут изменить значение этой переменной и сломать поведение этого класса